modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18051,6 +18051,9 @@ the-site-will-use-the-default-x-system-page-from-now-on.-are-you-sure-you-want-t
 the-sitemap-protocol-notifies-search-engines-of-the-structure-of-the-website=The sitemap protocol notifies search engines of the structure (sitemap) of the website.
 the-sku-could-not-be-found=The SKU could not be found.
 the-small-image-file-could-not-be-saved=The small image file could not be saved.
+the-sort-was-created=The sort was created.
```

Comment:
@cgmonte You could have created a generic language key, same thing for filter, but we can improve this later :)